Library/Formula/qbittorrent.rb

```diff
@@ -0,0 +1,36 @@
+class Qbittorrent < Formula
+  desc "Qt based free and reliable P2P Bittorrent client"
+  homepage 'http://www.qbittorrent.org'
+  url 'https://github.com/qbittorrent/qBittorrent/archive/release-3.2.3.tar.gz'
+  sha256 "816d611a0af50c469f52946c94b88c003f908ee031ee0e55a45273e4c64897c5"
+  head 'https://github.com/qbittorrent/qBittorrent.git',
+    :branch => ENV['QBT_BRANCH'] || "master"
+
+  depends_on "pkg-config" => :build
+  depends_on "qt" => :build
+  depends_on "libtorrent-rasterbar" => :build
+  depends_on "boost" => :build
+  depends_on "gnu-sed" => :build
+
+  resource "geoIP" do
+    url "http://geolite.maxmind.com/download/geoip/database/GeoLiteCountry/GeoIP.dat.gz"
```

Comment:
I was thinking about just running a cron job that monitors if the file changes and then automatically update the formula. But I see that if a user doesn't update formula, then they are screwed. I can't just keep a copy on github and distribute it due to their licensing. Other option is to keep an outdated copy in the code so we don't have to worry about adding it using formula.